tests/unit/conftest.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+
+from relay.blockchain.currency_network_events import (
+    TransferEventType,
+    TrustlineRequestEventType,
+    TrustlineUpdateEventType,
+)
+
+
+@pytest.fixture()
+def web3_event():
+    return {
+        "blockNumber": 5,
+        "transactionHash": "0x1234",
+        "address": "0x12345",
+        "logIndex": 0,
+        "blockHash": "0x123456",
+    }
+
+
+@pytest.fixture()
+def web3_event_trustline_update(web3_event):
+    web3_event.update(
+        {
+            "args": {
+                "_creditor": "0x123",
+                "_debtor": "0x1234",
+                "_creditlineGiven": 50,
+                "_creditlineReceived": 100,
+                "_isFrozen": True,
+            },
+            "event": TrustlineUpdateEventType,
+        }
+    )
+    return web3_event
+
+
+@pytest.fixture()
+def web3_event_trustline_request(web3_event):
```

Comment:
It could lead to issue if you want to use two events in the same test, because we update the event here. just as a note.

Reply:
.copy() it is!